dev-java/xml-commons-external/xml-commons-external-1.4.01-r1.ebuild

```diff
@@ -4,6 +4,7 @@
 EAPI=5
 
 JAVA_PKG_IUSE="doc source"
+MAVEN_ID="xml-apis:xml-apis:1.4.01"
```

Comment:
this commit i won't merge. i'd like to see it last-rited instead. this package is long time obsolete.

Reply:
Commit has been removed